packages/graph/hash_graph/lib/graph/src/knowledge/mod.rs

```diff
@@ -19,12 +19,12 @@ use crate::{
     store::query::Expression,
 };
 
-/// Query to read [`Entities`], which satisfy the [`Expression`].
+/// Query to read [`Entities`] or [`Link`]s, which satisfy the [`Expression`].
 ///
 /// [`Entities`]: Entity
 #[derive(Debug, Deserialize, Component)]
 #[serde(deny_unknown_fields, rename_all = "camelCase")]
-pub struct EntityQuery {
+pub struct KnowledgeQuery {
```

Comment:
Maybe call it `KnowledgeGraphQuery`?

Reply:
Applied in ee984bdab2943a042988130478e2fedce93e30d4